CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - The Conjur version is now printed on server startup, after running `conjurctl server`
   ([cyberark/conjur#1590](https://github.com/cyberark/conjur/pull/1590))
 
+### Security
+- Upgrade Rails to `v5.2.4.3` to resolve [CVE-2020-8164](https://groups.google.com/forum/#!topic/rubyonrails-security/f6ioe4sdpbY).
+- Use a new base image ruby-fips-base-image-ubuntu and ruby-fips-base-image-phusion
+  The new image consists of openssl 1.0.2u which is FIPS complaint 
```

Comment:
Would this be more of a change than a security fix?  What about moving this the the `Changes` section, maybe rephrasing it to something like:
```
- Uses OpenSSL 1.0.2u to support FIPS compliance.
```
?

Reply:
Done